tests/acceptance/api_test.go

```diff
@@ -1001,6 +996,13 @@ func TestApi(t *testing.T) {
 							},
 							f: func(r *testutil.R, apiResp *apiResponse) {
 								assert.Equal(r, 201, apiResp.StatusCode)
+
+								// Ignore error from this, since reset might have happened
```

Comment:
I think this comment wasn't written correctly to begin with. The issue is that reset _has not_ completed